src/components/map/Map.tsx

```diff
@@ -157,131 +144,264 @@ export const Map = ({
     };
   }
 
+  useEffect(() => {
+    if (clusteringEnabled) {
+      const index = new Supercluster({
+        radius: clusterDistance,
+        maxZoom: 20
+      });
+
+      const points = locations.map((location, index) => ({
+        type: 'Feature',
+        properties: { cluster: false, id: location.id, index },
+        geometry: {
+          type: 'Point',
+          coordinates: [location.position.longitude, location.position.latitude]
+        }
+      }));
+
+      index.load(points);
+      const bounds = [-180, -85, 180, 85];
```

Comment:
how come these exact values?

Reply:
the values represent the maximum possible latitude and longitude values for the earth's coordinate system, defining a rectangular area that covers the entire world.

but you're right, in our case it's simply not needed and might also improve performance slightly.
In the upcoming commit I've changed it to `const bounds = [2, 46, 18, 56];` to cover germany and parts of countries around. 
- Westernmost longitude: 2° E (covering parts of France and the Netherlands)
- Easternmost longitude: 18° E (covering parts of Poland and the Czech Republic)
- Northernmost latitude: 56° N (covering parts of Denmark)
- Southernmost latitude: 46° N (covering parts of Switzerland and Austria)

Reply:
and what was the outcome? any performance improvements?

Reply:
is the bounds the reason for fading other parts of the map that are outside? you can see what i mean when zoomed fully out seeing the whole world.

Reply:
no, the bounds are just there for clustering, it just takes in the mapmarker inside the bounds into account for clustering. 
that fade out you mean has todo with our `.mbtiles` map file cause we haven't got the data to display the whole world.